.github/dependabot.yml

```diff
@@ -4,7 +4,7 @@ updates:
     registries: "*"
     directory: "/"
     schedule:
-      interval: weekly
+      interval: monthly
```

Comment:
@thomasrichner-oviva do you prefer a more frequent interval as it was before, given this is a public repo?

Reply:
No, monthly should be fine. Ones with vulnerabilities anyways pop up earlier.